src/Gang/Gang.ts

```diff
@@ -149,15 +149,14 @@ export class Gang {
     gangFaction.playerReputation +=
       (Player.mults.faction_rep * respectGainsTotal * favorMult) / GangConstants.GangRespectToReputationRatio;
 
-    if (!(this.wanted === 1 && wantedLevelGainPerCycle < 0)) {
+    if (this.wanted !== 1 || wantedLevelGainPerCycle >= 0) {
       const oldWanted = this.wanted;
-      let newWanted = oldWanted + wantedLevelGainPerCycle * numCycles;
-      newWanted = newWanted * (1 - justice * 0.001); // safeguard
+      const newWanted = oldWanted + wantedLevelGainPerCycle * numCycles;
+      // Ensure that even members with poor stats can effectively lower wanted level
+      this.wanted = newWanted * (1 - justice * 0.001);
+      this.wantedGainRate -= newWanted - this.wanted;
       // Prevent overflow
-      if (wantedLevelGainPerCycle <= 0 && newWanted > oldWanted) newWanted = 1;
-
-      this.wanted = newWanted;
-      if (this.wanted < 1) this.wanted = 1;
+      if (this.wanted < 1 || (wantedLevelGainPerCycle <= 0 && this.wanted > oldWanted)) this.wanted = 1;
```

Comment:
What's the purpose of this extra condition? When would we be gaining negative wanted per cycle, but this.wanted would end up greater than the old wanted level?

Reply:
Oh I see that now, you just combined the two existing conditions for setting wanted to 1.

I'm not sure which edge case that is accounting for but I guess just leave it in. This looks fine to me.